### PR TITLE
Add meta tag in the header of HTML

### PR DESCRIPTION
**Pull request by tako-black**
refs #4267

### CODE REVIEW

File: docs/docs/tutorial.md [hunk 1]
```diff
@@ -220,6 +221,7 @@ First, add the third-party library **marked** to your application. This is a Jav
 ```html{7}
```
Comment (zpao) [inline docs/docs/tutorial.md]: Need to bump this if you're adding a line so that the correct line stays highlighted.

Comment (tako-black) [inline docs/docs/tutorial.md]: I'm sorry I can't understand what you mean.
What should I change?

Comment (zpao) [inline docs/docs/tutorial.md]: The 7 should become an 8. We use the `{7}` to highlight a line in the generated markup (see it in action @ http://facebook.github.io/react/docs/tutorial.html#adding-markdown)

Comment (tako-black) [inline docs/docs/tutorial.md]: Ah, thanks! I understand!
I've changed that.

**Comment by jimfb** [comment]
Looks great, thanks!

**Pull request merged**